### DISCUSSION TITLE
About the question regarding "Refresh ALL."

### DISCUSSION THREAD

**Discussion by zhaisixiang** [SideStore/SideStore#362] [Q&A]
In-app, running "Refresh ALL" works fine, and the experience is excellent! However, if using a shortcut to run "Refresh ALL Apps," although they refresh simultaneously, SideStore itself completes the refresh first. This prevents other apps from automatically refreshing through the shortcut. Have you encountered this issue?

**Comment by steilerDev** [comment]
I'm seeing the exact opposite, apps are refreshed but SideStore itself does not seem to refresh 🤔

**Reply by Elbozo123** [reply]
Same here
